next/components/paragraph--accordion--two-columns.tsx

```diff
@@ -0,0 +1,70 @@
+import clsx from "clsx";
+
+import { FormattedText } from "@/components/formatted-text";
+import { HeadingParagraph } from "@/components/heading--paragraph";
+import { Paragraph } from "@/components/paragraph";
+import { LinkButtonWithStyles } from "@/components/paragraph--hero";
+import { Accordion as ParagraphAccordion } from "@/lib/zod/paragraph";
+import ArrowIcon from "@/styles/icons/arrow-down.svg";
+
+import { Accordion } from "@/wunder-component-library/accordion";
+
+export function ParagraphAccordionTwoColumns({
+  paragraph,
+}: {
+  paragraph: ParagraphAccordion;
+}) {
+  return (
+    <>
+      {paragraph.field_heading && (
+        <HeadingParagraph>{paragraph.field_heading}</HeadingParagraph>
+      )}
+
+      <div className="grid grid-cols-1 md:grid-cols-2 md:gap-2">
+        <div>
+          {paragraph.field_formatted_text && (
+            <div className="flex items-center justify-center">
+              <FormattedText
+                html={paragraph.field_formatted_text.processed}
+                className={clsx(
+                  "text-left text-md/xl text-scapaflow sm:text-lg",
+                  paragraph.field_heading && "mt-4"
+                )}
+              />
+            </div>
+          )}
+          {paragraph.field_primary_link && (
+            <div className="flex items-center justify-center py-3">
+              <LinkButtonWithStyles
+                href={paragraph.field_primary_link.full_url}
+                className="text-base mr-4 inline-flex max-w-sm px-5 py-3"
+                variant="primary"
+              >
+                {paragraph.field_primary_link.title}
+                <ArrowIcon
+                  aria-hidden
+                  className="bg:white ml-3 h-6 w-6 -rotate-90"
```

Comment:
This `bg:white` class isn't a thing (would be bg-white), let's remove it

Reply:
I agree with Josh and I look forward to updating the variants/LInkButtonWithStyles soon once we have all current PRs merged.

Reply:
removed in : e0369eab4a9dd0238368cecebe4089ac7028b480 (it was in all the places where we use that arrowIcon component